config/runtimes/kserve-tensorflow-serving.yaml

```diff
@@ -3,6 +3,8 @@ kind: ClusterServingRuntime
 metadata:
   name: kserve-tensorflow-serving
 spec:
+  annotations:
```

Comment:
the only one I haven't figured out - docs are a little confusing. seems we should use --rest_api_port (8080 for us) https://www.tensorflow.org/tfx/serving/serving_config  but when I hit that with /metrics or /monitoring/prometheus/metrics , I get a `404`

Reply:
Did you try 8501? See https://stackoverflow.com/questions/52975301/prometheus-for-tensorflow-serving

Reply:
yes